[VCPKG] create .. failed

example

`vcpkg create draco-lanl https://github.com/lanl/Draco/archive/ffeff6b112924488c58024cab181ed6629a1ddc6.zip`


```
CMake Error at scripts/ports.cmake:15 (message):
          Your vcpkg executable is outdated and is not compatible with the current CMake scripts.
          Please re-build vcpkg by running bootstrap-vcpkg.
```


`vcpkg version`

Vcpkg package management program version 2019.09.12-nohash-external

additional options do not help

`vcpkg create --vcpkg-root E:\tools\vcpkg ...`

`set VCPKG_ROOT=E:\tools\vcpkg`

--- a/toolsrc/src/vcpkg/commands.create.cpp
+++ b/toolsrc/src/vcpkg/commands.create.cpp
@@ -24,7 +24,7 @@ namespace vcpkg::Commands::Create
 
         const fs::path& cmake_exe = paths.get_tool_exe(Tools::CMAKE);
 
-        std::vector<System::CMakeVariable> cmake_args{{"CMD", "CREATE"}, {"PORT", port_name}, {"URL", url}};
+        std::vector<System::CMakeVariable> cmake_args{{"CMD", "CREATE"}, {"PORT", port_name}, {"URL", url}, {"VCPKG_ROOT_PATH", paths.root}};
 
         if (args.command_arguments.size() >= 3)
         {

